src/components/SomePage.spec.tsx

```diff
@@ -17,23 +17,22 @@ describe("<SomePage/>", () => {
     expect(button).not.toBeDisabled();
   });
 
-  it("test 2", async () => {
-    const user = userEvent.setUp();
+  it("test 4", async () => {
     render(
       <SomePage/>
     );
     const button = screen.getByText("Click me");
-    user.click(button);
+    await userEvent.click(button);
     expect(button).not.toBeDisabled();
   });
-  
-  it("test 3", async () => {
-    const user = userEvent.setUp();
+
+  it("test 5", async () => {
     render(
       <SomePage/>
     );
+    const input = screen.getByTestId("input");
     const button = screen.getByText("Click me");
-    user.dblClick(button);
+    userEvent.type("Hello", input);
```

Comment:
@coderabbitai ensure calls to userEvent.type are always wrapped in a call to waitFor